docs/install-kafka-operator.md

```diff
@@ -1,28 +1,27 @@
 ---
-title: Install the Kafka operator
+title: Install the operator
 shorttitle: Install
 weight: 10
 ---
 
 
 
-The operator installs the 2.5.0 version of Apache Kafka, and can run on Minikube v0.33.1+ and Kubernetes 1.15.0+.
+The operator installs version 3.1.0 of Apache Kafka, and can run on Minikube v0.33.1+ and Kubernetes 1.19.0+.
 
-> The operator supports Kafka 2.0+
+> The operator supports Kafka 2.6.2-3.1.x.
```

Comment:
**Question**: wouldn't it be better to parametrize these floating versions for easier future update?

Reply:
I still think it would be easier to see these references at one place without scanning the docs themselves, but to my understanding it would have to be modified in the internal koperator docs first.